datum/umbrel-app.yml

```
@@ -0,0 +1,26 @@
+manifestVersion: 1.1
+id: datum
+category: Bitcoin
```

This just needs to be all lowercase:
```suggestion
category: bitcoin
```